spring-data-mongodb/src/main/java/org/springframework/data/mongodb/observability/MongoMetricsConfigurationHelper.java

```diff
@@ -0,0 +1,21 @@
+package org.springframework.data.mongodb.observability;
+
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import io.micrometer.tracing.Tracer;
+
+import com.mongodb.client.SynchronousContextProvider;
+
+/**
+ * Helper functions to ease registration of Spring Data MongoDB's observability.
+ */
+public class MongoMetricsConfigurationHelper {
+
+	public static SynchronousContextProvider synchronousContextProvider(Tracer tracer, ObservationRegistry registry) {
+		return () -> new SynchronousTraceRequestContext(tracer).withObservation(Observation.start("name", registry));
```

Comment:
If we don't have an `Observation.start()` somewhere, then nothing seems to get logged in Zipkin. This is what I had used in the past inside the ZipkinIntegrationTests, but it feels quite arbitarary.

If there is a better place to "start" things, I would like to know and could adjust things accordingly.